openpype/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -28,12 +28,36 @@ class CreateTextures(Creator):
     icon = "picture-o"
 
     default_variant = "Main"
+    image_format = None
+    exportPresetUrl = None
+    exportSize = None
+    exportPadding = "infinite"
+    exportDilationDistance = 16
```

Comment:
There's no need to store these as class attributes right? Especially because now it's changing for each created instance down below.